app/components/conferences/admin/SponsorForm.js

```diff
@@ -0,0 +1,365 @@
+                                </Row>
+                                <Form.Group className="mb-3">
+                                    <Form.Label>Name</Form.Label>
+                                    <Form.Control
+                                        required
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        type="text"
+                                        name="name"
+                                        placeholder="Name"
+                                        defaultValue={spon.attributes.name}
+                                    />
+                                </Form.Group>
+                                <Form.Group className="mb-3">
+                                    <Form.Label>Description</Form.Label>
+                                    <Form.Control
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        name="description"
+                                        as="textarea"
+                                        type="textarea"
+                                        placeholder="A short sweet biography"
+                                        defaultValue={spon.attributes.description}
+                                    />
+                                </Form.Group>
+                                <Form.Group className="mb-3">
+                                    <Form.Label>URL</Form.Label>
+                                    <Form.Control
+                                        type="url"
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        name="url"
+                                        placeholder="https://link-to.image"
+                                        defaultValue={spon.attributes.url}
+                                    />
+                                </Form.Group>
+                                <Form.Group className="mb-3">
+                                    <Form.Label>Logo-Url</Form.Label>
+                                    <Form.Control
+                                        type="url"
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        name="logo-url"
+                                        placeholder="https://link-to.image"
+                                        defaultValue={spon.attributes["logo-url"]}
+                                    />
+                                </Form.Group>
+                                <InputGroup className="mb-3">
+                                    <InputGroup.Text>Level</InputGroup.Text>
+                                    <Form.Control
+                                        type="number"
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        name="level"
```

Comment:
We should not allow negative levels. WDYT?

Reply:
There was no such clarification in the Api documentation but I also think that the levels should be 0 or more than 0, So I have added that condition in the form.